lib/metrics.py

```diff
@@ -327,12 +328,19 @@ def _get_db_conn(self):
             self.db = None
 
         if not self.db:
+            # get_database config may return None or empty
             db_config = database_config.get_database_config()
+            if db_config is None or "DatabaseType" not in db_config:
```

Comment:
As above, you probably want `if not db_config` here.

Reply:
changed